session_items.py

```diff
@@ -76,9 +100,24 @@ def delete_item_by_id(id) -> None:
     Args:
         item: The item to delete.
     """
-    current_items = session['items']
+    url = 'https://api.trello.com/1/cards/'+id+request_credentials
+
+    response = requests.delete(url)
+
+    print(response.text)
+
+def get_lists() -> dict:
+    json_response = json.loads(requests.get(base_request_url+'lists'+request_credentials).content)
```

Comment:
slightly shorter than `json.loads(response.content)` is to just use `response.json()`